docs/getting-started.md

```diff
@@ -225,27 +224,26 @@ newgrp docker
 ```
 
 You can install and run all the programs as Docker containers by using the
-[gravitino-playground](https://github.com/datastrato/gravitino-playground). For the details of
-how to run the playground, please see [how-to-use-the-playground](./how-to-use-the-playground.md)
+[gravitino-playground](https://github.com/datastrato/gravitino-playground). For details about
+how to run the playground, see [how-to-use-the-playground](./how-to-use-the-playground.md)
 
 ## Installing Gravitino playground locally
 
-The same steps apply to installing the playground locally as on AWS or Google Cloud Platform. You
-can follow the instructions for [Installing Gravitino playground on AWS or Google Cloud Platform](#installing-gravitino-playground-on-aws-or-google-cloud-platform).
+The same steps for installing the playground on AWS or Google Cloud Platform apply when installing it locally. Follow [Installing Gravitino playground on AWS or Google Cloud Platform](#installing-gravitino-playground-on-aws-or-google-cloud-platform).
 
 ## Using REST to interact with Gravitino
 
-After starting the Gravitino distribution, issue REST commands to create and modify metadata. While you are using localhost in these examples, run these commands remotely via a hostname or IP address once you establish correct access.
+After starting the Gravitino distribution, issue REST commands to create and modify metadata. While you are using `localhost` in these examples, run these commands remotely via a hostname or IP address once you establish correct access.
 
-1. Create a Metalake
+1. Create a Metalake:
 
    ```shell
    curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
    -H "Content-Type: application/json" \
    -d '{"name":"metalake","comment":"Test metalake"}' http://localhost:8090/api/metalakes
    ```
 
-   Verify the MetaLakes creation
+   Verify the MetaLake's creation:
```

Comment:
We probably should be consistent with Metalake vs MetaLake. I think we've mostly been using Metalake.